release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -849,6 +849,31 @@ Helm-based Operator enhancements include:
 {product-title} now properly supports the `terminationGracePeriodSeconds`
 parameter with the CRI-O container runtime.
 
+[discrete]
+[id="ocp-4-5-readyz-config"]
+==== /readyz configuration for API server health probe
+
+All {product-title} 4.5 clusters must be configured to use the `/readyz`
+endpoint for API server health checking to remain supported. Any clusters
+installed on versions prior to {product-title} 4.5 must be reconfigured to use
+`/readyz`.
+
+Clusters using user-provisioned infrastructure without `/readyz` configured can
+suffer from API outages when the API server restarts. This can be caused by
+events such as configuration changes, certificate updates, or master node
+reboots.
+
+The load balancer should be configured to take a maximum of 30 seconds from the
```

Comment:
What load balancer? If it's the one that you need to provide for a UPI cluster, I might merge this paragraph with the previous one.

Rewrite to avoid "should"
> ! should v
Always prefer a more direct verb or way of saying something. Omit “should,” or use “must” or “might,” as appropriate.